qml/pages/OverviewPage.qml

```diff
@@ -349,7 +333,7 @@ Page {
                                         spacing: Theme.paddingSmall
                                         Text {
                                             id: chatListLastUserText
-                                            text: (typeof display.last_message !== "undefined") ? ( display.last_message.sender_user_id !== overviewPage.ownUserId ? Emoji.emojify(Functions.getUserName(tdLibWrapper.getUserInformation(display.last_message.sender_user_id)), font.pixelSize) : qsTr("You") ) : qsTr("Unknown")
+                                            text: last_message_sender_id ? ( last_message_sender_id !== overviewPage.ownUserId ? Emoji.emojify(Functions.getUserName(tdLibWrapper.getUserInformation(last_message_sender_id)), font.pixelSize) : qsTr("You") ) : qsTr("Unknown")
```

Comment:
This seems to change the behavior for channels implicitly. Now it displays "Unknown" instead of an empty string. Not a real blocker, in a later change I will make the text invisible if it's belonging to a channel....

Reply:
Right, good catch. Too many "Unknown"s just litters the display. I wonder if this "Unknown" was needed at all.

Reply:
It was a failsafe string from the very beginning. However, it seems that it's only really needed for conversations with deleted users.